File: csharp_dotnetcore/51.Enterprise-Bot-Template/Middleware/Telemetry/LuisTelemetryConstants.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace EnterpriseBot
{
    /// <summary>
    /// The Application Insights property names that we're logging.
    /// </summary>
    public static class LuisTelemetryConstants
    {
        public const string IntentPrefix = "LuisIntent";  // Application Insights Custom Event name (with Intent)
        public const string IntentProperty = "Intent";
        public const string IntentScoreProperty = "IntentScore";
        public const string ConversationIdProperty = "ConversationId";
        public const string QuestionProperty = "Question";
        public const string ActivityIdProperty = "ActivityId";
        public const string SentimentLabelProperty = "SentimentLabel";
        public const string SentimentScoreProperty = "SentimentScore";
    }
}

File: samples/javascript_typescript/52.enterprise-bot/src/middleware/telemetry/luisTelemetryConstants.ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License

/**
 * The Application Insights property names that we're logging.
 */
export class LuisTelemetryConstants {
    public static readonly IntentPrefix: string = "LuisIntent";  // Application Insights Custom Event name (with Intent)
    public static readonly IntentProperty: string = "Intent";
    public static readonly IntentScoreProperty: string = "IntentScore";
    public static readonly ConversationIdProperty: string = "ConversationId";
    public static readonly QuestionProperty: string = "Question";
    public static readonly ActivityIdProperty: string = "ActivityId";
    public static readonly SentimentLabelProperty: string = "SentimentLabel";
    public static readonly SentimentScoreProperty: string = "SentimentScore";
}
